### ISSUE TITLE
Fix PyTorch LTS install instructions

### ISSUE THREAD

**Issue by pmeier** [pmeier/light-the-torch#55]
The instructions changed from 

```
pip install torch==1.11.0+cu113 -f https://download.pytorch.org/whl/cu113/torch_stable.html
```

to

```
pip install torch --extra-index-url https://download.pytorch.org/whl/cu113
```

It seems the exact specification of the version is no longer needed. Plus, I'm guessing this change or whatever it entailed also led to the ROCm wheels being detected. This is disabled in #54, but should be revisited.

**Comment by pmeier** [comment]
With pytorch/pytorch#74753 merged, we can probably switch our backend to this style.

**Comment by pmeier** [comment]
Everything but LTS was fixes in #59.

**Comment by pmeier** [comment]
Although pytorch/pytorch/pull/74753 is merged now, as of 16.04.2022 the official installation instruction still use the `-f` flag. 

In addition, the new links seem to be not set up properly. Without a trailing `/` the LTS link is broken whereas this works for all other channels:

- https://download.pytorch.org/whl/lts/1.8 vs. https://download.pytorch.org/whl/lts/1.8/
- https://download.pytorch.org/whl/nightly vs. https://download.pytorch.org/whl/nightly/

The behavior is the same if we append a specific computation backend:

- https://download.pytorch.org/whl/lts/1.8/cpu vs. https://download.pytorch.org/whl/lts/1.8/cpu/

Let's wait until this is fixed on their side before we also migrate.

**Comment by pmeier** [comment]
This is now tracked in pytorch/pytorch#76829.
